src/app/modules/core-bcp/components/facility-number/facility-number.component.html

```diff
@@ -6,7 +6,7 @@
          [(value)]='facNumber'
          (change)='inputChange($event)'
          (blur)='onBlur($event)'
-         bcpValidateFacilityNumber
+         appBcpValidateFacilityNumber
```

Comment:
Revert this change, we shouldn't need "app" to prefix our directives.

If there's a rule requiring it, we can take a look at removing it.

I thought (but may be mistaken) that the directive prefix comes from the angular.json file for the application name prefix (line 10).  But if a lint required you to change this, then I must be wrong.  If there's a specific tslint rule in tslint.json that requires all direcives start with app we can just remove that one. It seems like a silly rule to me but I'm open to having my mind changed.

Reply:
Reverted this change, and removed the `'app'` rule from `tslint.json`.